js/resourceInfo.js

```diff
@@ -27,45 +27,66 @@ var licences = {
     "CC-BY-SA-3.0": { "name": "CC BY-SA 3.0", "url": "https://creativecommons.org/licenses/by-sa/3.0/" },
     "CC-BY-NC-ND-3.0": { "name": "CC BY-NC-ND 3.0", "url": "https://creativecommons.org/licenses/by-nc-nd/3.0/" },
     "OFL": { "name": "OFL", "url": "http://scripts.sil.org/OFL" },
+    "Freepik": { "name": "Freepik terms of use", "url": "https://www.freepikcompany.com/legal#nav-freepik" }
 }
 
 var resources = [
             {
                 "description": qsTr("Background"),
-                "name": "Rice paper 2",
-                "url": "http://subtlepatterns.com/rice-paper-2/",
-                "author": "Atle Mo",
-                "authorUrl": "http://atle.co/",
-                "licence": "CC-BY-SA-3.0",
-                "additionalInfo": ""
+                "info" :
+                [{
+                    "name": "Rice paper 2",
+                    "url": "http://subtlepatterns.com/rice-paper-2/",
+                    "author": "Atle Mo",
+                    "authorUrl": "http://atle.co/",
+                    "licence": "CC-BY-SA-3.0",
+                    "additionalInfo": ""
+                }]
             },
             {
                 "description": qsTr("Decorative"),
-                "name": "Ivy",
-                "url": "https://pixabay.com/pl/ivy-wirowa%C4%87-upadek-design-okr%C4%85g-303546/",
-                "author": "Clker-Free-Vector-Images",
-                "authorUrl": "https://pixabay.com/pl/users/Clker-Free-Vector-Images-3736/",
-                "licence": "CC0",
-                "additionalInfo": qsTr("Colors changed")
+                "info" :
+                [{
+                    "name": "Ivy",
+                    "url": "https://pixabay.com/pl/ivy-wirowa%C4%87-upadek-design-okr%C4%85g-303546/",
+                    "author": "Clker-Free-Vector-Images",
+                    "authorUrl": "https://pixabay.com/pl/users/Clker-Free-Vector-Images-3736/",
+                    "licence": "CC0",
+                    "additionalInfo": qsTr("Colors changed")
+                }]
             },
             {
                 "description": qsTr("Icons"),
-                "name": "Devine",
-                "url": "http://findicons.com/pack/1579/devine/1",
-                "author": "ipapun",
-                "authorUrl": "http://ipapun.deviantart.com/",
-                "licence": "CC-BY-NC-ND-3.0",
-                "additionalInfo": ""
+                "info" :
+                [{
+                    "name": "Devine",
+                    "url": "http://findicons.com/pack/1579/devine/1",
+                    "author": "ipapun",
+                    "authorUrl": "http://ipapun.deviantart.com/",
+                    "licence": "CC-BY-NC-ND-3.0",
+                    "additionalInfo": ""
+                },
+                {
+                    "name": "flaticon",
```

Comment:
You should provide a name of the real icon or icon set you used and the url to the same.
In your case it would be probably: Pencil Edit Button / https://www.flaticon.com/free-icon/pencil-edit-button_61456

Reply:
Done